org.mapeditor.Tiled.json

```diff
@@ -18,9 +18,31 @@
     "/share/thumbnailers"
   ],
   "modules": [
+    {
+      "name": "qbs",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.qt.io/official_releases/qbs/1.20.0/qbs-src-1.20.0.tar.gz",
+          "sha256": "44961a4bb61580ae821aaf25ebb5a5737bd8fb79ec0474aa2592cdd45cc5171f"
+        },
+        {
+          "type": "script",
+          "dest-filename": "configure",
+          "commands": [
+            "qmake QBS_INSTALL_PREFIX=/app CONFIG+=qbs_enable_project_file_updates qbs.pro"
+          ]
+        }
+      ]
+    },
     {
       "name": "tiled",
-      "buildsystem": "qmake",
+      "buildsystem": "simple",
+      "build-commands": [
+        "qbs setup-toolchains --detect",
+        "qbs --no-install qbs.installPrefix:/app",
+        "qbs install --no-build --install-root /"
+      ],
       "config-opts": [
         "CONFIG+=release"
```

Comment:
This line will no longer have any effect when using Qbs. Probably instead `config:release` should be added to the last two build commands:
```
        "qbs --no-install config:release qbs.installPrefix:/app",
        "qbs install --no-build --install-root / config:release"
```

Reply:
Oops! #12 should fix this